refmt.el

```diff
@@ -201,8 +203,19 @@ function."
            (if (zerop (let* ((files (list (list :file outputfile) errorfile))
                              (args (append width-args (list "--parse" from "--print" to bufferfile))))
                         (cond ((equal refmt-command 'opam)
-                               (apply 'call-process
-                                      "opam" nil files nil (append '("exec" "--" "refmt") args)))
+                               (if refmt-opam-bin-dir
+                                   (apply 'call-process (concat refmt-opam-bin-dir "/refmt") nil files nil args)
+
+                                 (progn
+                                   (setq-local refmt-opam-bin-dir (with-temp-buffer
+                                                                    (when (eq (call-process-shell-command
+                                                                               "opam config var bin" nil (current-buffer) nil) 0)
+                                                                      (let ((bin-path
```

Comment:
I don't understand why you need to assign to a variable before to return the path